tests/rspec/lib/cluster.rb

```
@@ -192,8 +193,10 @@ def wait_for_bootstrapping
     puts 'HOORAY! The cluster is up'
   rescue
     ips.each do |master_ip|
-      ['bootkube', 'tectonic', 'kubelet', 'k8s-node-bootstrap'].each do |s|
-        print_service_logs(master_ip, s)
+      save_docker_logs(master_ip, @name)
+
+      ['bootkube', 'tectonic', 'kubelet', 'k8s-node-bootstrap'].each do |service|
+        print_service_logs(master_ip, service, @name)
       end
     end
     raise
```

Here you can raise the exception `e` you caught in line 194.

---

done